addons/crm/models/crm_lead.py

```diff
@@ -2147,11 +2149,27 @@ def _pls_get_naive_bayes_probabilities(self, batch_mode=False):
         To avoid this, we add 0.1 in each frequency. With few data, the computation is than not really realistic.
         The more we have records to analyse, the more the estimation will be precise.
         :return: probability in percent (and integer rounded) that the lead will be won at the current stage.
+
+        If is_tooltip is True, method instead recomputes the probability of self, that should be a singleton, and instead
+        returns a dict containing team name, probability, and a list of all (score, field, value) triplets for all value
```

Comment:
Would be great to have a documented example :)

Reply:
There is one in the test with values. But I can add a structure example in the dosctring if you want?